integration-testing-config.yaml

```diff
@@ -0,0 +1,15 @@
+components:
+  bblfshd:
+    port: 9433
+
+  bblfsh_web:
+    port: 8082
+
+  gitbase_web:
+    port: 8081
```

Comment:
8081 is the default port for bblfsh_web, so this may also collide.

Reply:
right 🤦‍♂️

Reply:
fixed